service-front/module/Application/view/layout/layout.twig

```diff
@@ -40,8 +40,7 @@
     </div>
 </header>
 <div class="govuk-width-container">
- {%  include 'layout/phase_banner.twig' %}
-    <main class="govuk-main-wrapper" id="main-content" role="main">
+    <main class="govuk-main-wrapper-custom" id="main-content" role="main">
```

Comment:
Rather than adding a custom class, you can use a spacing utility class ([documentation](https://design-system.service.gov.uk/styles/spacing/#static-spacing)):
```suggestion
    <main class="govuk-main-wrapper govuk-!-padding-bottom-7" id="main-content" role="main">
```